docs/website/blog/2022-06-14-binding-external-service.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Binding an external service with odo v3
+author: Philippe Martin
+author_url: https://github.com/feloy
+author_image_url: https://github.com/feloy.png
+tags: ["binding"]
+slug: binding-external-service-with-odo-v3
+---
+
+How to bind an external service using odo v3
+
+<!--truncate-->
+
+When developers are working on a micro-service that needs to access a database or another service, 
+they may want to provide to their application the address and the necessary credentials to access
+this service as simply as possible.
+
+In this article, we are saying that we are *binding* the service to the application.
+
+Using the Service Binding Operator and creating some Kubernetes resources for
+each service you want to be bindable, you can make the life easier for developers.
+
+## Creating a Service resource to redirect to the external service
+
+To expose an external service from inside a Kubernetes cluster, you can create a *Headless* Service,
+and manually create the Enpoints to access this external service.
+
+Here an example, to connect to an external Redis service on IP 192.168.1.10 and port 6379:
```

Comment:
```suggestion
Here is an example, to connect to an external Redis service on IP 192.168.1.10 and port 6379:
```

Also, a more realistic example would be using an external DNS name, no? Maybe you could use an example with an external domain name instead, or mention this as well..